src/native-stack/views/NativeStackView.tsx

```diff
@@ -100,6 +100,7 @@ const MaybeNestedStack = ({
       ]}
       // @ts-ignore Wrong props passed to View
       stackPresentation={stackPresentation}
+      collapsable={false}
```

Comment:
Why we need to set collapsable to false here? 🤔

Reply:
Good idea

Reply:
https://github.com/software-mansion/react-native-screens/pull/1825/commits/037a34813660bfa4db7880dd66968cf350c74c3d